src/lib/bootloader/grub2_widgets.rb

```diff
@@ -497,8 +497,6 @@ def init
       Yast::UI.ChangeWidget(Id(:gfxmode), :Value, mode) if mode && mode != ""
 
       Yast::UI.ChangeWidget(Id(:theme), :Value, grub_default.theme || "")
-    # FIXME: just temporary workaround for terminal that does contain
-    # more complex string (bsc#1053559)
```

Comment:
you remove only comment, but keep catching of exception which expect GRUB_TERMINAL which is wrong.

Reply:
I've added support for parsing multiple values for that option ... so the exception means something different now.

Reply:
OK, so maybe it is time to do it properly and define own exception class only for this case and catch this specialized class? When I did this workaround in past, I try to do minimal changes, but when you touched cfa already, it makes more sense for me.